files/default/event_handler/write_chef_error_handler.rb

```diff
@@ -0,0 +1,14 @@
+
+module WriteChefError
+  class WriteChefError < Chef::Handler
```

Comment:
It's worth adding a comment here to describe what this class is doing, and when someone should use it.

Reply:
Thanks!